### PR TITLE
change loglevel to debug

### PR DESCRIPTION
**Pull request by leodotcloud**

### CODE REVIEW

**Comment by ibuildthecloud** [comment]
Is the intention to not print the token?  Can we just drop the token, or make printing the token only at debug level.  This message is useful to know when things aren't connecting as it repeats.  The logging is so light in the agent I'd hate to have even less information.

**Comment by leodotcloud** [comment]
>Is the intention to not print the token?

yes

@ibuildthecloud updated the PR with requested changes.

File: cmd/agent/main.go [hunk 1]
```diff
@@ -331,7 +331,8 @@ func run() error {
 		if !isConnect() {
 			wsURL += "/register"
 		}
-		logrus.Infof("Connecting to %s with token %s", wsURL, token)
+		logrus.Infof("Connecting to %s", wsURL)
```
Comment (deniseschannon) [inline cmd/agent/main.go:334]: Could we do a substring of the actual token as part of the info statement?

**Comment by snasovich** [comment]
Replaced by https://github.com/rancher/rancher/pull/33042

**Pull request closed without merging**